acceptance/framework/vault/vault_cluster.go

```diff
@@ -122,7 +112,9 @@ func (v *VaultCluster) SetupVaultClient(t *testing.T) *vapi.Client {
 		tunnel.Close()
 	})
 
-	config.Address = fmt.Sprintf("http://127.0.0.1:%d", localPort)
+	config.Address = fmt.Sprintf("https://127.0.0.1:%d", localPort)
+	err := config.ConfigureTLS(&vapi.TLSConfig{Insecure: true})
```

Comment:
Sanity check : is `Insecure: true` correct?

Reply:
This is for local traffic, and so we don't need to verify TLS for that.

Reply:
Ah! makes sense, thanks!